packages/dd-trace/src/config.js

```diff
@@ -236,6 +237,25 @@ function reformatSpanSamplingRules (rules) {
 
 class Config {
   constructor (options = {}) {
+    // Read & apply the YAML configuration
```

Comment:
All of this new code could be in a method instead of right in the constructor.

Reply:
Indeed, I just migrated it 👍